CHANGELOG.md

```diff
@@ -28,6 +28,9 @@ _None_
 * Strings dot-syntax template: use `enum`s for namespacing instead of `struct`s.  
   [David Jennes](https://github.com/djbe)
   [#218](https://github.com/AliSoftware/SwiftGen/pull/218)
+* Added a `strings-no-comments-swift3` template that does not include the default translation of each key.  
+  [Loïs Di Qual](https://github.com/ldiqual)
+  [#222](https://github.com/AliSoftware/SwiftGen/issues/222)
```

Comment:
I've just released 4.0.1 (while your PR is not part of it) and then merged master into your branch to update your PR… but now your CHANGELOG entry seems to have been merged in the wrong CHANGELOG section :wink: whoops.

Could you please move it to the new `### New Features` subsection **of the `## master` section**?

Thx!